odoo_backup_sh/models/odoo_backup_sh.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Stanislav Krotov <https://it-projects.info/team/ufaks>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+import json
+import werkzeug
+
+import odoo
+from odoo import http, api, fields, models
+from odoo.http import request
+from odoo.addons.odoo_backup_sh.controllers.main import BackupController, BACKUP_SERVICE_ENDPOINT
+from odoo.addons.iap import jsonrpc, InsufficientCreditError
+
+
+class Dashboard(models.Model):
+    _name = 'odoo_backup_sh.dashboard'
+    _description = 'Backup Dashboard'
+
+    @api.model
+    def action_dashboard_redirect(self):
+        return self.env.ref('odoo_backup_sh.backup_dashboard').read()[0]
```

Comment:
what is ``.read()[0]``? Can you just use ``.id``?

Do you need this method besides ``ir.actions.server``? If not, you could this move there 
``action = env.ref('odoo_backup_sh.backup_dashboard')...``

Reply:
.read()[0] returns record dict mapping field names to their values, like this
{'create_uid': (1, 'Administrator'), 'xml_id': 'odoo_backup_sh.backup_dashboard', 'context': '{}' ...}
with .id odoo raises attribute error

- "Do you need this method besides ir.actions.server?"
I think it's a good way to manage action views in the future

Reply:
Ok, odoo uses the same approach